sigs.yaml

```diff
@@ -3697,6 +3697,44 @@ workinggroups:
     liaison:
       github: saschagrunert
       name: Sascha Grunert
+- dir: wg-node-lifecycle
+  name: Node Lifecycle
+  mission_statement: >
+    Explore and improve node and pod lifecycle in Kubernetes. This should result in
+    better node drain/maintenance support and better pod disruption/termination. It
+    should also improve node and pod autoscaling, better application migration and
+    availability, load balancing, de/scheduling, node shutdown, cloud provider integrations,
+    and support other new scenarios and integrations.
+
+  charter_link: charter.md
+  stakeholder_sigs:
```

Comment:
You are missing network, that owns loadbalancing, endpoints, ... and already has serious problems because of known issues like https://docs.google.com/document/d/1t25jgO_-LRHhjRXf4KJ5xY_t8BZYdapv7MDAxVGY6R8/edit?tab=t.0#heading=h.i4lwa7rdng7y